content/cybersecurity/concepts/malware/terms/trojan/trojan.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: 'Trojan'
+Description: 'Trojan is a form of downloaded malware that disguises itself as a legitimate application so it can trick users into downloading it, thus taking control of over their computer and other devices'
```

Comment:
```suggestion
Description: 'A trojan is a form of malware that disguises itself as a legitimate application so it can trick users into installing it, thus taking control of over their computer and other devices.'
```